README.md

```diff
@@ -1,2 +1,61 @@
 # marketo-api-php
 Unofficial Marketo API PHP library.
+
+# Installation
+
+## Composer
+
+In your project's `composer.json` add to `repositories` section the following:
+
+```json
+{
+  "type": "vcs",
+  "url": "git@github.com:cra/marketo-api-php.git"
```

Comment:
Shouldn't url have `cra-repo` instead of `cra`?

Reply:
Fixed.